src/solvers/problem_description/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+# other Serac Project Developers. See the top-level LICENSE file for
+# details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+set(problem_description_headers
```

Comment:
I think I like `physics_utilities` for this folder as `problem` is almost as overloaded as `solver`!

Reply:
While we're at it, we could rename the `solvers` directory to `physics_modules`.

Reply:
Another subdirectory for `physics_operators` would also be useful.

Reply:
it could just be named "utilities" since it is already "physics_modules/utilities".

Reply:
For that matter it, it could just be `physics` then `physics/operators` and `physics/utilities`.